src/components/Lottie/index.js

```diff
@@ -0,0 +1,3 @@
+import Lottie from './Lottie';
```

Comment:
Why `Lottie.tsx` is written in TS and `index.js` in JS? I think Lottie catalog can be removed and `Lottie.tsx` file in `components` catalog should be sufficient.

@ZhenjaHorbach

Reply:
> Why `Lottie.tsx` is written in TS and `index.js` in JS?

This is already existing pattern, i.e. MapView but agree to write TS for all

> I think Lottie catalog can be removed and `Lottie.tsx` file in `components` catalog should be sufficient.

Agree. Unless platform specific or has propTypes in js